docs/API.md

```diff
@@ -1,39 +1,36 @@
+The library uses [`require-in-the-middle`](https://www.npmjs.com/package/require-in-the-middle)
+and [`shimmer`](https://www.npmjs.com/package/shimmer) to instrument your Node.js application.
 
-The SignalFx-Tracing Library for JavaScript provides auto-instrumentation for all its supported libraries and frameworks using [`require-in-the-middle`](https://www.npmjs.com/package/require-in-the-middle) and [`shimmer`](https://www.npmjs.com/package/shimmer).  To utilize this functionality, the tracer must be accessed and initialized before any target library or framework is imported:
+For information about steps for the express configuration to automatically instrument an application,
+see [SignalFx Tracing Library for JavaScript](/README.md).
 
-```javascript
-// init() invocation must occur before importing any traced library (e.g. Express)
-const tracer = require('signalfx-tracing').init(
-  // service name, also configurable via
-  // SIGNALFX_SERVICE_NAME environment variable
-  service: 'my-traced-service',
-  // Smart Agent or Gateway endpoint, also configurable via
-  // SIGNALFX_ENDPOINT_URL environment variable
-  url: 'http://my_agent_or_gateway:9080/v1/trace', // http://localhost:9080/v1/trace by default
-  // optional organization access token, also configurable via
-  // SIGNALFX_ACCESS_TOKEN environment variable
-  accessToken: 'myOptionalOrganizationAccessToken'
-)
-
-// auto-instrumented Express application
-const express = require('express') 
-const app = express()
-```
+## Manually instrument a Node.js application
 
-### Custom Instrumentation
+In addition to instrumentation the SignalFx Tracing Library for JavaScript
+provides, you can add custom instrumentation to your application with the
+[OpenTracing API](#opentracing-api) and [Scope](#scope) manager.
 
-Regardless if you are using a [supported library instrumentation](#instrumentations), you may want to add custom instrumentation to your code.  This can be done using the [OpenTracing API](#opentracing-api) and [Scope](#scope) utility.
+Because a number of helpful methods aren't provided by the current OpenTracing
+JavaScript API, the delegation pattern of the OpenTracing global tracer
+(`require('opentracing').globalTracer()`) isn't fully compatible with our tracer
+instances. As a result, you need to reference the actual tracer instance
+returned by `init()` where custom instrumentation utilizes scope management
+and manual trace reporting.
 
-**Note**: Because a number of helpful methods aren't provided by the current OpenTracing JavaScript API, the delegation pattern of the OpenTracing global tracer (`require('opentracing').globalTracer()`) isn't fully compatible with our tracer instances.  For this reason, a reference to the actual tracer instance returned by `init()` should be made accessible where custom instrumentation utilizes scope management and manual trace reporting.
+### Use the OpenTracing API with the SignalFx Tracing Library for JavaScript
```

Comment:
Given the content of the previous paragraph this seems out of order or now unnecessary.

Reply:
the previous paragraph is an intro to the next couple headings